.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,30 @@
+name: Static code analysis
+
+on: push
+
+jobs:
+
+  flake8:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+
+      - run: |
+          python -m pip install --upgrade pip
+          python -m pip install flake8  flake8-import-order flake8-blind-except flake8-builtins
```

Comment:
I don't recall having to install anything other than `flake8` in our other repos?

Reply:
The add-on packages add specific checks not covered by flake8. I copied from [grfn-distribution](https://github.com/asfadmin/grfn-distribution/blob/test/.github/workflows/static-analysis.yml), but we use the same add-on packages in other repos like [hyp3](https://github.com/ASFHyP3/hyp3/blob/develop/requirements-all.txt) and [asf_tools](https://github.com/ASFHyP3/asf-tools/blob/develop/.github/workflows/static-analysis.yml).

That does point out that dropping `flake8-blind-except` (for now) is probably a better interim solution than adding `noqa` to the three blind excepts currently in the code.